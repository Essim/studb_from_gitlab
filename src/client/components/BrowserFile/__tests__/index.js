import React from 'react';
import {BrowserFiles} from '../';
import { shallowToJson } from 'enzyme-to-json';

const mockFiles= {
        _contents:[]
    },
    mockName = "name",
    mockOnClick = jest.fn(),
    mockDoc_cuid="mockCUID",
mockSelectFilePreviewed=jest.fn();
describe('BrowserFiles', () => {
    it('should render correctly', () => {
        const output = shallow(<BrowserFiles files={mockFiles} name={mockName} onclick={mockOnClick} doc_cuid={mockDoc_cuid} selectFilePreviewed={mockSelectFilePreviewed}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
