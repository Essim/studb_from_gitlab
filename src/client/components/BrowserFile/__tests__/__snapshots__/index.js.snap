// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowserFiles should render correctly 1`] = `
<NestedFileTreeView
  directory={
    Object {
      "Fichiers": Object {
        "_contents": Array [],
      },
      "_contents": Array [],
    }
  }
  fileClickHandler={[Function]}
  fileTemplate={[Function]}
  folderTemplate={[Function]}
/>
`;
